skunkworks_crow/src/main/java/org/odk/share/views/ui/instance/fragment/ReviewedInstancesFragment.java

```diff
@@ -224,7 +224,7 @@ public void sendForms() {
                 instanceIds.add(transferInstance.getInstanceId());
             }
         }
-        Intent intent = new Intent(getContext(), SendActivity.class);
+        Intent intent = new Intent(getContext(), HpSenderActivity.class);
```

Comment:
There should be a dialog asking for one of the options out of the `Bluetooth` and `Hotspot`, here it is directly triggering hotspot.

Reply:
Sorry for forgetting that, thanks for reminding. 
Fixed.